src/extension/_locales/de/messages.json

```diff
@@ -630,6 +633,27 @@
   "user_switch": {
     "message": "Benutzer/in wechseln"
   },
+  "v7_context_menu": {
+    "message": "Installieren Sie die neue AEM Sidekick"
+  },
+  "v7_hint_cover": {
+    "message": "https://www.aem.live/tools/sidekick/en/v7-teaser.png"
+  },
+  "v7_hint_description": {
+    "message": "Eine brandneue Version der AEM Sidekick-Erweiterung ist jetzt im Chrome Web Store verfügbar. Entscheiden Sie sich noch heute für verbesserte Leistung, neue Funktionen und ein nahtloses Authoring-Erlebnis."
+  },
+  "v7_hint_title": {
+    "message": "Die neue AEM Sidekick"
```

Comment:
Use masculine when referring to the sidekick in german.

Reply:
@rofe these should be improved once we get vendor human translation (to replace MT)